assets/js/base/components/payment-methods/payment-method-options.js

```diff
@@ -42,50 +45,50 @@ const PaymentMethodOptions = () => {
 	const { removeNotice } = useStoreNotices();
 	const { isEditor } = useEditorContext();
 
+	const options = Object.keys( paymentMethods ).map( ( name ) => {
+		const { edit, content, label, supports } = paymentMethods[ name ];
+		const component = isEditor ? edit : content;
+		return {
+			value: name,
+			label:
+				typeof label === 'string'
+					? label
+					: cloneElement( label, {
+							components: paymentMethodInterface.components,
+					  } ),
+			name: `wc-saved-payment-method-token-${ name }`,
+			content: (
+				<PaymentMethodTab allowsSaving={ supports.savePaymentInfo }>
+					{ cloneElement( component, {
+						activePaymentMethod,
+						...paymentMethodInterface,
+					} ) }
+				</PaymentMethodTab>
+			),
+		};
+	} );
```

Comment:
This might be something that could be optimized in a performance followup, although it's likely relatively quick to execute whenever there are render calcs.